workgroup/overrides.json

```diff
@@ -12,15 +13,21 @@
                 "Value": "tagValue2"
             }
         ],
-        "/WorkGroupConfiguration/ResultConfiguration/OutputLocation": "s3://bucket",
-        "/WorkGroupConfiguration/ResultConfiguration/EncryptionConfiguration/KmsKey": "abcdef54hi",
-        "/WorkGroupConfiguration/ResultConfiguration/ExpectedBucketOwner": "123456789012",
+        "/WorkGroupConfiguration/ResultConfiguration/OutputLocation": "s3://ramkomma-bucket",
```

Comment:
These values should be fixed. I think we can remove this file, since we are in V2 now.